lib/ssllabs/api.rb

```diff
@@ -18,7 +18,7 @@ def initialize
 
     def request(name, params = {})
       name = name.to_s.camelize(:lower)
-      uri = URI("#{API_LOCATION}#{name}?#{params.to_query}")
+      uri = URI("#{API_LOCATION}#{name}?#{params.collect { |k,v| "#{k}=#{v}" }.join("&")}")
```

Comment:
This will not uri-encode k and v properly:

```
irb(main):003:0> params = {"foo" => "/"}
=> {"foo"=>"/"}
irb(main):005:0> URI("/?#{params.to_query}").to_s
=> "/?foo=%2F"
irb(main):006:0> URI("/?#{params.collect { |k,v| "#{k}=#{v}" }.join("&")}").to_s
=> "/?foo=/"
```